From a983b6bbdcdf659ce1029c2def74cc3b326ce62b Mon Sep 17 00:00:00 2001
From: WangJing <wangjing@mprc.pku.edu.cn>
Date: Wed, 12 Oct 2011 19:44:09 +0800
Subject: [PATCH 019/641] UniCore64: Add asm/io.h

Fixed error:
kernel/resource.c:28:9: error: 'IO_SPACE_LIMIT' undeclared here (not in a function)
mm/bootmem.c: In function 'init_bootmem_core':
mm/bootmem.c:101:2: error: implicit declaration of function 'phys_to_virt'
lib/devres.c: In function 'devm_ioremap_release':
lib/devres.c:8:2: error: implicit declaration of function 'iounmap'

Signed-off-by: WangJing <wangjing@mprc.pku.edu.cn>
---
 arch/unicore64/include/asm/Kbuild |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/unicore64/include/asm/Kbuild b/arch/unicore64/include/asm/Kbuild
index 05ca0f9..92ab916 100644
--- a/arch/unicore64/include/asm/Kbuild
+++ b/arch/unicore64/include/asm/Kbuild
@@ -21,7 +21,6 @@ generic-y += ftrace.h
 generic-y += futex.h
 generic-y += hardirq.h
 generic-y += hw_irq.h
-generic-y += io.h
 generic-y += ioctl.h
 generic-y += ioctls.h
 generic-y += ipcbuf.h
-- 
1.7.9.5

